Makefile

```diff
@@ -170,7 +170,8 @@ $(LUA):
 
 	scripts/module_check.sh
 
-	[ -e openwrt/.config ] || $(OPENWRTMAKE) defconfig
+	[ -e openwrt/.config ] || $(GLUON_ENV) scripts/basic_openwrt_config.sh > openwrt/.config
+	$(OPENWRTMAKE) defconfig
```

Comment:
If we're going to do an unconditional `defconfig` anyways here, we could also ensure that the `AUTOREMOVE` config is always up-to-date regardless of old `.config` contents by doing:
```sh
$(GLUON_ENV) scripts/basic_openwrt_config.sh >> openwrt/.config
$(OPENWRTMAKE) defconfig
```
(without checking for `.config` existence first)